### PR TITLE
v18.0

### PR DESCRIPTION
**Pull request by a3s7p**
- remove webshell link from webcp
- remove outdated comment
- overlay: update openvpn scripts wrt upstream changes
- shellcheck lints
- simplify TLS init steps
- add mod_accesslog to lighttpd
- update delexpired regex to match HTTP/2 too
- cp instead of mv for crl.pem
- update changelog

### CODE REVIEW

File: overlay/etc/lighttpd/conf-available/50-tklcp.conf [hunk 1]
```diff
@@ -1,3 +1,4 @@
+server.modules += ( "mod_accesslog" )
```
Comment (JedMeister) [inline overlay/etc/lighttpd/conf-available/50-tklcp.conf:1]: Good catch man. Although we should probably add this to common and/or all appliances too I suspect!? Thoughts?

**Review by JedMeister — COMMENTED** [review]
Looks good man.

Although I did notice that there is a issue (feature request) that may be worth looking at? See https://github.com/turnkeylinux/tracker/issues/1347. After having a closer look, I've added an updated comment to the OP of that issue.

File: overlay/etc/lighttpd/conf-available/50-tklcp.conf (hunk 1 above)
Comment (a3s7p) [reply overlay/etc/lighttpd/conf-available/50-tklcp.conf:1]: lighttpd already ships with a catch-all access log config, it just has to be enabled which should be a single line change in `common/conf/lighttpd`. i will prepare the PR

Comment (a3s7p) [reply overlay/etc/lighttpd/conf-available/50-tklcp.conf:1]: https://github.com/turnkeylinux/common/pull/292

**Pull request merged**